hw/ip/rv_dm/data/rv_dm.hjson

```diff
@@ -102,13 +102,14 @@
   registers: {
     regs:[
     ]
-    rom: [
-      // ROM size (given as `items` below) must be a power of two.
+    mem: [
+      // Debug memory size (given as `items` below) must be a power of two.
       { window: {
-          name: "ROM"
+          name: "MEM"
           items: "1024" // 4 KiB
-          swaccess: "ro",
-          desc: '''Access window into the debug ROM.'''
+          swaccess: "rw",
+          byte-write: "true",
```

Comment:
this is the relevant part of the change. the remaining diffs are only due to the name change.

Reply:
was the previous attribute actually doing anything inside `rv_dm`..? i can't remember. Or was this more for the testbench?

Reply:
No, the RTL was not affected, but the RAL model uses this for predictions.

Reply:
sounds good.